src/main/scala/latis/service/hapi/AboutResponse.scala

```diff
@@ -0,0 +1,53 @@
+package latis.service.hapi
+
+import io.circe.Encoder
+
+/**
+ * Response from the `/about` endpoint.
+ *
+ * @param version version of HAPI
+ * @param status HAPI status object
+ * @param id unique ID for the server
+ * @param title human-readable name for the server
+ * @param contact contact info for server issues
+ * @param description brief description of the type of data served
+ * @param contactId ID in the discovery system for info about the contact
+ * @param citation how to cite the server
+ */
+final case class AboutResponse(
+  version: String,
+  status: Status,
+  id: String,
+  title: String,
+  contact: String,
+  description: String = "",
+  contactId: String = "",
+  citation: String = ""
+)
+
+object AboutResponse {
+
+  /** JSON encoder */
+  implicit val encoder: Encoder[AboutResponse] =
+    //TODO: don't include empty optional params
```

Comment:
I'd mark the optional values as `Option[String]` rather than using empty strings.

Circe will encode `None` to `null`, which usually isn't what we want either, but there's an easy way to drop the `null` values:

```scala
Encoder.forProduct8(...)(...).mapJson(_.dropNullValues)
```

This will produce an encoder that accepts `Option` but doesn't keep them in the JSON if they are `None`.

Reply:
I couldn't just tack `.mapJson(_.dropNullValues)` onto the end of what was already there because it wouldn't compile, but it worked great after I factored out the anonymous function into its own `val`.